supply-chain-security/compromises/compromise-definitions.md

```diff
@@ -0,0 +1,39 @@
+# Types of Supply Chain Compromise
+
+This document serves to provide a consistent set of definitions, recognized by SIG-Security, for the Supply Chain Compromise Catalog.  The types of compromise may be expanded in the future as new or more distinct compromises occur.  When this happens, these definitions will be updated.
+
+
+**Source Code Compromise** - occurs when source code repositories (public or private) is manipulated intentionally by the developer or through a developer or repository credential compromise.  Source Code compromised can also occur when source code in open-source dependencies is manipulated in a manner to make them more useful.
```

Comment:
>manipulated in a manner to make them more useful.

Hmm, what dose it mean by more useful here?

Reply:
nit: for consistency, the other definitions should follow "X compromise"

Reply:
updated to include an explanation of more useful.  Not everything is technically a compromise in the definitions.  For example - Negligence is not a compromise.  One could argue that given the lexicons in use elsewhere, "compromise" should be dropped from the title of this definition and be left at "source code".  Thoughts?

Reply:
> One could argue that given the lexicons in use elsewhere, "compromise" should be dropped from the title of this definition and be left at "source code". Thoughts?

Yea - i agree with this!

Reply:
fixed